.github/workflows/pytest.yml

```diff
@@ -29,12 +29,12 @@ jobs:
         python-version: [ 3.6, 3.7, 3.8 ]
         test-markers: [ "not distributed", "distributed" ]
         include:
-          - python-version: 3.6
-            tensorflow-version: 2.3.1
           - python-version: 3.7
             tensorflow-version: 2.5.0
           - python-version: 3.8
-            tensorflow-version: 2.5.0
+            tensorflow-version: 2.7.0
+          - python-version: 3.9
```

Comment:
Need to change `python-version` above to `[ 3.7, 3.8, 3.9 ]`

Reply:
Done.